io.github.letslearn-app.yaml

```
@@ -0,0 +1,56 @@
+app-id: io.github.letslearn.letslearn-app
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
```

I mentionned the path have to be updated
```suggestion
  append-path: /usr/lib/sdk/node16/bin
```